### ISSUE TITLE
Feature: Broadcasting room

### ISSUE THREAD

**Issue by Gummikavalier** [strukturag/nextcloud-spreedme#45]
I'm not sure if WebRTC supports efficiently this, but it would be nice to have a room type that lets the user, who created the room, to broadcast video and audio only one way. The rest of the users would **not** trigger negotiations of audio/video back to broadcaster. Chat would be allowed.

Would be handy in presentations for big amount of people.

**Comment by brtptrs** [comment]
I support this idea. It would be very useful

**Comment by cguentherTUChemnitz** [comment]
I think it should be slightly more generalized to not fix this to the person who created the room. This should be switchable during the broadcasting session. For example big development teams, having a strategy discussion. There you have the typically one screensharing guy, who presents stuff. When his part of the discussion is done, than he want to switch the screensharing to another person.

The interesting part is, in this constellation only one person provides a stream to a lot of people. If this person is on a business trip in an hotel with weak wireless, than the whole session is doomed, because the webrtc peer-to-peer connection enforces him to provide the screen sharing stream to all guests. I think some kind of media relay would be very feasible in this situation. So the screen sharing guy has only to provide one stream, which is multiplied to the others using such a media relay with good internet connection.

**Comment by aostrovsky** [comment]
Yeah. It could be good to be able drop p2p and delegate media role to something on server.
